internal/plans/internal/planproto/planfile.pb.go

```diff
@@ -517,14 +517,14 @@ type Change struct {
 	// the documentation for any message that embeds Change.
 	Action Action `protobuf:"varint,1,opt,name=action,proto3,enum=tfplan.Action" json:"action,omitempty"`
 	// msgpack-encoded HCL values involved in the change.
-	// - For update and replace, two values are provided that give the old and new values,
-	//   respectively.
-	// - For create, one value is provided that gives the new value to be created
-	// - For delete, one value is provided that describes the value being deleted
-	// - For read, two values are provided that give the prior value for this object
-	//   (or null, if no prior value exists) and the value that was or will be read,
-	//   respectively.
-	// - For no-op, one value is provided that is left unmodified by this non-change.
+	//   - For update and replace, two values are provided that give the old and new values,
```

Comment:
These changes to the protobuf generated code are reverted when you run `make protobuf`. I guess we'll need to either update the proto files to match gofmt…?

I'm also surprised we don't have a check tracking that `make protobuf` results in no changes.

Reply:
Along the way here I initially _didn't_ run `make protobuf` and got an error from our "quick checks" workflow, so what you see here is the result of me running `make protobuf` after everything's upgraded to Go 1.19 versions in order to make tat pass.

Can you share what you tried that led you to seeing an error? I'm trying to understand if there's just something in your environment that isn't upgraded yet or if there's a more annoying inconsistency to deal with here. :thinking:

Reply:
The "Quick Checks" workflow is running everything all in a single `make` command like this:

```bash
make fmtcheck importscheck generate staticcheck exhaustive protobuf
```

I wonder if the problem here is that `make fmtcheck` and `make protobuf` are "fighting" one another by both expecting different results, and the check thing happens to be running them in the right order for my code here to pass. :thinking:

Reply:
Oh, oops, my mistake! I wasn't using go 1.19 in the terminal tab where I ran `make protobuf`, and confirm that it executes cleanly when I do.